src/components/file-upload/FileList.tsx

```diff
@@ -23,6 +44,22 @@ function FileList({ files, onDelete }: NewType) {
             </Avatar>
           </ListItemAvatar>
           <ListItemText primary={file.name} />
+          <FormControl>
+            <InputLabel id="choose-type-label">Избери тип</InputLabel>
+            <Select
+              labelId="choose-type-label"
+              id="choose-type"
+              value={(filesRole.find((f) => f.file === file.name)?.role as string) || 'Избери тип'}
+              label="Избери тип"
+              onChange={setFileRole(file)}>
+              <MenuItem value="Избери тип">Избери тип</MenuItem>
+              <MenuItem value="background">background</MenuItem>
+              <MenuItem value="coordinator">coordinator</MenuItem>
+              <MenuItem value="campaignPhoto">campaignPhoto</MenuItem>
+              <MenuItem value="invoice">invoice</MenuItem>
+              <MenuItem value="document">document</MenuItem>
```

Comment:
Let's use an enum here and loop over the values/keys

Adding these manually will be considered bad practice as we might forget to do it

Reply:
On master branch we have those values in a enum already

https://github.com/podkrepi-bg/frontend/blob/77540771be860821daf1cbabcc7867263cf32cd3/src/components/campaign-file/roles.ts#L1-L9